modules/loki-rate-limit-errors.adoc

```diff
@@ -0,0 +1,83 @@
+// Module is included in the following assemblies:
+// * logging/cluster-logging-loki.adoc
+// * logging/cluster-logging-external.adoc
+
+:_content-type: PROCEDURE
+[id="loki-rate-limit-errors_{context}"]
+= Troubleshooting Loki rate limit errors
+
+If the Log Forwarder API forwards a large block of messages to Loki that exceeds the rate limit, Loki generates rate limit (`429`) errors.
+
+These errors can occur during normal operation. For example, when adding {ocp-logging} to a cluster that already has some logs, rate limit errors might occur while {ocp-logging} tries to ingest all of the existing log entries. In this case, if the rate of addition of new logs is less than the total rate limit, the historical data is eventually ingested, and the rate limit errors are resolved without requiring user intervention.
```

Comment:
Again I would suggest removing the comma before "and" (last sentence of the paragraph)